composer/core/evaluator.py

```diff
@@ -96,6 +110,13 @@ class Evaluator:
             If a callable, it should take two arguments (:class:`.State`, :class:`.Event`) and return a bool
             representing whether the evaluator should be invoked. The event will be either :attr:`.Event.BATCH_END`
             or :attr:`.Event.EPOCH_END`.
+
+            When specifying ``eval_interval``, the evaluator(s) are also run at the ``Event.FIT_END`` if it doesn't
+            evenly divide the training duration.
+
+        eval_at_fit_end (bool, optional):
```

Comment:
To match the trainer signature, can we require that the `eval_interval`  incorporate the `eval_at_fit_end` flag? (E.g. if you want to set it to False, pass in `eval_interval=evaluate_periodically(..., eval_at_fit_end=False)`?

Reply:
Done!